src/dune

```diff
@@ -1,4 +1,4 @@
 (library
  (name claudius)
  (public_name claudius)
- (libraries tsdl))
+ (libraries tsdl uutf))
```

Comment:
Why is this change here? As far as I can tell it's because of the changes to font.ml, which in turn aren't needed.